lib/src/command/update_flutter_sdk_command.dart

```diff
@@ -29,24 +28,23 @@ class UpdateFlutterSdkCommand extends MoronepoCommand<Null> {
   @override
   FutureOr<Null> run() async {
     final rootDirectory = moronepoResults.workingDirectory ?? Directory.current.path;
-    final finder = ProjectFinder();
-    Iterable<Project> projects = await finder.find(path: rootDirectory);
-    final project = projects.firstWhere((it) => it.isRoot, orElse: () => null);
+    final projects = await ProjectFinder().find(path: rootDirectory);
+    final rootProject = projects.firstWhere((it) => it.isRoot, orElse: () => null);
```

Comment:
What do you think about moving find root responsibility to `ProjectFinder`? It can be found much faster than iterate recursively every folder and then iterate thru founded projects.